Report for GSOC 2019.md

```diff
@@ -284,3 +288,9 @@ Weekly Activity
    - Timings for presentation of XrViz to Pangeo.
 2. Presented Xrviz to Pangeo in Weekly Meeting. A 10 minute demonstration of the interface.
 3. Completed [PR #37](https://github.com/intake/xrviz/pull/37). Now users can also visualise dask xarrays.
+
```

Comment:
I think it would be nice to have a new section just before the `Acknowledgments` section called "Moving forward" or  "Project status" or something to indicate that while this is the end of the summer of code project, it's the *beginning* of the actual software package you have created, and that it is well positioned to grow and thrive, embedded in the intake project, documented, etc.    Perhaps move these sentences from the `Acknowledgements`:
> Working with ESIP, Intake and Pyviz under Google Summer of Code has been an amazing experience. This has given me a chance to learn more about creating interactive dashboards, python best practices, earth science, software development, open source and much more. I feel the project is in good state and would be extremely useful for the community.

and expand a bit.

Reply:
Thanks for this review! I have made changes accordingly.